build/common/installer/scripts/tomlparser.rb

```diff
@@ -219,6 +220,14 @@ def populateSettingValuesFromConfigMap(parsedConfig)
         end
       end
     end
+
+    #Get annotation based log filtering setting
+    begin
+      if !parsedConfig[:log_collection_settings][:filter_using_annotations].nil? && !parsedConfig[:log_collection_settings][:filter_using_annotations][:enabled].nil?
```

Comment:
do we want to set this if @logEnableKubernetesMetadata is false?

Reply:
refer to the reply of the previous comment